operators/multiclusterobservability/controllers/placementrule/manifestwork.go

```diff
@@ -498,20 +498,26 @@ func generateMetricsListCM(client client.Client) (*corev1.ConfigMap, error) {
 		Data: map[string]string{},
 	}
 
-	allowlist, err := getAllowList(client, operatorconfig.AllowlistConfigMapName)
+	allowlist, ocp3Allowlist, err := getAllowList(client, operatorconfig.AllowlistConfigMapName)
 	if err != nil {
 		log.Error(err, "Failed to get metrics allowlist configmap "+operatorconfig.AllowlistConfigMapName)
 		return nil, err
 	}
 
-	customAllowlist, err := getAllowList(client, config.AllowlistCustomConfigMapName)
+	customAllowlist, ocp3CustomAllowlist, err := getAllowList(client, config.AllowlistCustomConfigMapName)
```

Comment:
It will make the things complex that allows user customizing the allow list for OCP 3 and OCP 4. for the custom allow list, we do not need to differentiate the metrics is for OCP 3 or others, just apply to all. WDYT?

Reply:
Let's make it simpler and not differentiate the custom metrics.